packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx

```diff
@@ -307,6 +312,7 @@ class UserReviewFormComponent extends React.Component<
         </Button>
```

Comment:
@jpye-finch 
When creating / updating an user, we are checking if the user requires a signature or not from [`SIGNATURE_REQUIRED_FOR_ROLES[]`](https://github.com/opencrvs/opencrvs-farajaland/blob/e3253dd7a8907d869c67fa6ce3f120b43be6476c/src/api/application/application-config-default.ts#L55) . (How) does the check changes now that scopes are involved?